ui/src/data_explorer/components/QueryEditor.js

```diff
@@ -164,7 +164,7 @@ class QueryEditor extends Component {
       const start = this.editor.selectionStart
       const end = this.editor.selectionEnd
       const filteredTemplates = templates.filter(t =>
-        t.tempVar.includes(matched[0].substring(1))
+        t.tempVar.startsWith(matched[0])
```

Comment:
I actually committed with `indexOf` at first, and then remembered `startsWith`. Hence my question about whether we are supporting non-Chrome ;)